test/fixtures/render/content-type.apib

```diff
@@ -0,0 +1,14 @@
+# GET /
++ Response 200 (application/json; charset=utf-8)
+    + Attributes
+        + user: pksunkara
+
+# GET /hal
++ Response 200 (application/hal+json)
+    + Attributes
+        + user: pksunkara
+
+# GET /typo
++ Response 200 (application/haljson)
```

Comment:
Is this a real media type? Or do we know of any media type that is `application/whateverjson` ? My point is shouldn't we be checking for either `application/json` or `application/whatever+json` (I know the regex would be more complex 😜 )